packages/sheets/src/services/ref-range/util.ts

```diff
@@ -872,6 +873,24 @@ export const handleDeleteRangeMoveUpCommon = (param: IDeleteRangeMoveUpCommand,
     return queryObjectMatrix(matrix, (v) => v === 1);
 };
 
+export const handleRemoveRowCommon = (param: IRemoveRowColCommandInterceptParams, targetRange: IRange) => {
```

Comment:
The `Common` suffix confuses me whether there is a "not common" method?

And the function should have unit tests to ensure it is correctly implemented.